lib/typescript/httpclient/mappers/channelMapper.ts

```diff
@@ -4,5 +4,9 @@ import {ChannelApiPayload} from '../payload/ChannelApiPayload';
 export const channelMapper = (payload: ChannelApiPayload): Channel => ({
   ...payload,
   sourceChannelId: payload.source_channel_id,
+  metadata: {
+    ...payload.metadata,
+    imageUrl: payload.metadata.image_url,
```

Comment:
Nitpick: doesn't this mean you will end up with `image_url` and `imageUrl` in the data?

Reply:
Yes, that's a good point actually :/ The problem is that `metadata` can contain structures provided by the user. Ok so I will make the mapping here explicit for now and we figure out how to do this at some other point.